client/my-sites/marketing/connections/services/google-drive.js

```diff
@@ -0,0 +1,105 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { last, isEqual } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { deleteStoredKeyringConnection } from 'state/sharing/keyring/actions';
+import { SharingService, connectFor } from 'my-sites/marketing/connections/service';
+
+export class GoogleDrive extends SharingService {
+	static propTypes = {
+		...SharingService.propTypes,
+		deleteStoredKeyringConnection: PropTypes.func,
+	};
+
+	static defaultProps = {
+		...SharingService.defaultProps,
+		deleteStoredKeyringConnection: () => {},
+	};
+
+	createOrUpdateConnection = () => {};
+
+	/**
+	 * Deletes the passed connections.
+	 *
+	 * @param {Array} connections Optional. Connections to be deleted.
+	 *                            Default: All connections for this service.
+	 */
+	removeConnection = () => {
+		this.setState( { isDisconnecting: true } );
+		this.props.deleteStoredKeyringConnection( last( this.props.keyringConnections ) );
```

Comment:
See how `google_my_business` site keyrings are disconnected when a user triggers the "Disconnect" action:  https://github.com/Automattic/wp-calypso/blob/add/google-sheets-connection/client/my-sites/marketing/connections/services/google-my-business.js#L71